chp20_sound/example_20_07_envelope/sketch.js

```diff
@@ -8,7 +8,7 @@ var osc;
 
 var envelope;
 
-var scale = [
+var notebank = [
```

Comment:
```suggestion
var notes = [
```

How about just "notes"?